README.dev.md

```diff
@@ -0,0 +1,75 @@
+# Developer documentation
+
+- Fork this repository.
+- Clone your fork.
+- Add this repo as upstream with
```

Comment:
```suggestion
1. Fork this repository.
2. Clone your fork.
3. Add this repo as upstream with the following command:
```

Reply:
I would put is as an enumeration list to give it more of a workflow feeling, what do you think?